ci/jenkins/test.sh

```diff
@@ -125,6 +125,13 @@ function clean_antrea {
     done
 }
 
+function clean_antrea_images {
+    IMGS=("<none>" "base-ubuntu" "antrea-ubuntu" "flow-aggregator")
```

Comment:
maybe we should keep recent "base-ubuntu" images for the sake of caching, even if the image can be built quite quickly?

BTW, in other jobs I have been using `docker system prune --force --all --filter until=48h` to also take care of the docker cache. Could that work?

Reply:
I think yes and I will have a try.